src/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -168,27 +173,48 @@ public String getDescription() {
       // A small warning so users will not over-write a file,
       if (f.exists()) {
         final int choice =
-            JOptionPane.showConfirmDialog(null, "A file by that name already exists. Do you wish to over write it?",
+            JOptionPane.showConfirmDialog(frame, "A file by that name already exists. Do you wish to over write it?",
                 "Over-write?", JOptionPane.YES_NO_OPTION, JOptionPane.WARNING_MESSAGE);
         if (choice != JOptionPane.OK_OPTION) {
           return;
         }
       }
+
       final File file = f;
-      final Runnable t = () -> {
-        if (saveScreenshot(node, file, frame, gameData)) {
-          JOptionPane.showMessageDialog(null, "Map Snapshot Saved", "Map Snapshot Saved",
-              JOptionPane.INFORMATION_MESSAGE);
+      final SwingWorker<?, ?> task = new SwingWorker<Void, Void>() {
```

Comment:
To its own place. I think that after looking at the test case and seeing that we have to test UI logic to get to this component. To fix that test, testing the screenshot logic directly, using an interface for the UI component interactions I believe would 'fix' the test problem, so that we have a relatively clean isolated test. The UI testing then IMO can be done by hand, in theory it's all library code anyways.